backend/chart.go

```diff
@@ -36,20 +36,32 @@ func (backend *Backend) allCoinCodes() []string {
 	return allCoinCodes
 }
 
-// ChartData assembles chart data for all active accounts.
-func (backend *Backend) ChartData() (interface{}, error) {
-	type chartEntry struct {
-		Time  int64   `json:"time"`
-		Value float64 `json:"value"`
-	}
+// ChartEntry is one data point in the chart timeseries.
+type ChartEntry struct {
+	Time  int64   `json:"time"`
+	Value float64 `json:"value"`
+}
 
+// Chart has all data needed to show a time-based chart of their assets to the user.
+type Chart struct {
+	DataMissing bool         `json:"chartDataMissing"`
+	DataDaily   []ChartEntry `json:"chartDataDaily"`
+	DataHourly  []ChartEntry `json:"chartDataHourly"`
+	Fiat        string       `json:"chartFiat"`
+	Total       *float64     `json:"chartTotal"`
```

Comment:
might as well remove the pointer here? there's `DataMissing` for that. or is it also valid when data is missing?

Reply:
yeah, it is independent of DataMissing, but can also be missing :joy:  I will add comments to each field.